src/azure-cli-core/azure/cli/core/tests/test_util.py

```diff
@@ -270,20 +270,20 @@ def test_send_raw_requests(self, send_mock, get_raw_token_mock):
                          generated_client_request_id_name=None)
 
         get_raw_token_mock.assert_not_called()
-        request = send_mock.call_args.args[1]
+        request = send_mock.call_args[0][1]
```

Comment:
`args` and `kwargs` are not available in Python 3.6.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args

> Changed in version 3.8: Added `args `and `kwargs` properties.